.github/workflows/deploy.yml

```
@@ -0,0 +1,304 @@
+    
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Deploy to production servers
+      env:
+        PROD_HOSTS: ${{ secrets.PROD_HOSTS }}
+        PROD_USER: ${{ secrets.PROD_USER }}
+        PROD_KEY: ${{ secrets.PROD_SSH_KEY }}
+      run: |
+        echo "🚀 Deploying to production environment"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)